### ISSUE TITLE
Undo doesn't move focus

### ISSUE THREAD

**Issue by dejbug** [queler/holokenmod#30]
When undoing, it is difficult to see where the undo happened. The highlighted cell should guide the player's attention to what changes.

**Comment by dejbug** [comment]
Sorry, the highlight does move. The input focus doesn't, but it should

**Comment by queler** [comment]
Yep, the shading is still reflecting what the undo is modified.  The undo structure would have to be modified a bit
